src/components/Header/index.js

```diff
@@ -1,97 +1,88 @@
-import React, { useState } from 'react'
+import React, { Fragment, useState } from 'react'
 
 import styled from 'styled-components'
 import Flex from 'styled-flex-component'
 import { Padding } from 'styled-components-spacing'
+import breakpoint from 'styled-components-breakpoint'
+import remcalc from 'remcalc'
 
+import navLinks from './navLinks'
 import { Row, Col, Grid } from '../grid'
 import Hamburger from './Hamburger'
 import Overlay from './Overlay'
 import TopNav from './TopNav'
 import SideNav from './SideNav'
 
-const FixedHeightFlex = styled(Flex)`
-  height: 84px;
+const StyledPadding = styled(Padding)`
+  display: flex;
+  height: ${remcalc(120)};
+  align-items: center;
 `
 
-const navLinks = [
-  {
-    label: 'Services',
-    dropdownItems: [
-      {
-        label: 'Engineering',
-        to: '/engineering/'
-      },
-      {
-        label: 'Design',
-        to: '/design/'
-      },
-      {
-        label: 'Training',
-        to: '/training/'
-      }
-    ]
-  },
-  {
-    label: 'Our work',
-    to: '/our-work/'
-  },
-  {
-    label: 'Blog',
-    href: 'https://medium.com/yld-engineering-blog/'
-  },
-  {
-    label: 'About',
-    dropdownItems: [
-      {
-        label: 'Our team',
-        to: '/about-us/'
-      },
-      {
-        label: 'Contact',
-        to: '/contact/'
-      }
-    ]
-  },
-  {
-    label: 'Join us',
-    to: '/join-us/'
-  }
-]
+const StyledGrid = styled(Grid)`
+  position: fixed;
+  height: ${remcalc(84)};
+  background: ${props =>
+    props.theme.colors[props.isSpecialityPage ? 'blueBg' : 'white']};
+  width: 100%;
+  max-width: unset;
+  z-index: 100;
```

Comment:
I did not! Good shot @jsms90 !!! I updated with props.theme.zIndexes.header 👍 

By the way, it reminds me of something I saw in theme which I think is an absolute nonsense.
in ColorsVariables we have `darkBlack: '#1d1d1d'`, `black: '#333333'`, `white: '#fff'`
I have to use a real black (#000) in the ticket. How come we have defined black as `black: '#333333'` in theme if we have white as fff?? Should we make a ticket to update that and rename the 333333 black to something else and update where used accordingly? It's only a 5-10 minutes job but as they say at Tesco, every little help! @Daveawb thoughts?

Reply:
I find that, whenever you come into a codebase that has been written by other people before you, you will come across things that you don't agree with or wouldn't necessarily have done in the same way yourself. In moments like this, it's easy to be incredulous about the 'mess' you feel that you've been given. But I would always be mindful that the people before you may have had good reason to do the things that seem like 'absolute nonsense'.

So, although you are right. #333333 is technically grey, not black & I agree that the naming is perhaps misleading. But it's worth looking our [design documentation](https://app.zeplin.io/project/5b4e666f0d4230a16dffd93a/screen/5c61564dd958777e20e8faec). Here you can see that designers have referred to this colour as 'black'. Probably because visually speaking, it _looks_ black when compared with the lighter greys below.

This is presumably the reason why one of the developers has called it black in our codebase. Speaking a common language between design and engineering is important, after all. Perhaps whichever engineer wrote this, didn't have a designer to hand to suggest an alternative name & chose to go with common language over technical correctness.

I don't know :woman_shrugging: 

In this instance, if we want to change the name in the codebase, I would suggest having a conversation with one designers about updating the Zeplin docs too.

Reply:
I can be blunt sometimes but having white as `#fff` and black as not `#000` is just not making sense. You're right; there could be a proper reason behind it. I completely agree with that as a conclusion: we need to raise it with James, as actual designer and see if it was agreed for a reason or if we could change it.

Reply:
sounds good :+1: